### PR TITLE
libplist 2.3.0

### PR DESCRIPTION
**Pull request by p-linnane**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/libplist.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 class Libplist < Formula
   desc "Library for Apple Binary- and XML-Property Lists"
   homepage "https://www.libimobiledevice.org/"
-  url "https://github.com/libimobiledevice/libplist/archive/2.2.0.tar.gz"
-  sha256 "7e654bdd5d8b96f03240227ed09057377f06ebad08e1c37d0cfa2abe6ba0cee2"
-  license "LGPL-2.1"
+  url "https://github.com/libimobiledevice/libplist.git",
+      tag:      "2.3.0",
+      revision: "72480212cd8ec0ab8d0b5064df52c146cac603b4"
```
Comment (carlocab) [inline Formula/libplist.rb:6]: Doing a Git checkout just to generate the version is a bit too much. Let's keep the tarball and just set
```ruby
ENV["RELEASE_VERSION"] = version.to_s
```
in the `install` method before calling `autogen.sh` instead.

**Comment by carlocab** [comment]
`libimobiledevice` needs https://github.com/libimobiledevice/libimobiledevice/commit/eda2c5ea71829f11d69342e6858d09aa53943938, but the patch doesn't apply cleanly. Probably worth asking upstream if they'll cut a new release at some point.

**Comment by p-linnane** [comment]
Opened an issue upstream: https://github.com/libimobiledevice/libimobiledevice/issues/1435

**Comment by reitermarkus** [comment]
This also contains a breaking change for the `plist_from_memory` function, which breaks compilation for `shairport-sync`, see https://github.com/mikebrady/shairport-sync/issues/1674.

We don't seem to build it with AirPlay 2 support (`--with-airplay-2`) though.

**Pull request closed without merging**